src/database/DataTypes/TemperatureBreach.js

```diff
@@ -7,10 +7,6 @@ import Realm from 'realm';
 import moment from 'moment';
 
 export class TemperatureBreach extends Realm.Object {
-  get colour() {
```

Comment:
Just double checking if this line is a potential reference?  https://github.com/openmsupply/mobile/blob/c21dedf5f31bd667ab284dff76608adfc8241b2f/src/widgets/BreachChart.js#L22

`colour` doesn't ring a bell to me though and looks like it's usually `null`...unless configured in desktop somewhere 🤔 ?

Reply:
it is indeed.
@wlthomson would you care to comment? Would be nice to wrap up this PR

Reply:
Well `this.temperatureBreachConfiguration?.colour ?? null` is always going to be `undefined` (as `temperatureBreachConfiguration` is not a field. I might be missing something, but not sure how desktop config would make a difference? Are you suggesting just replace the method with `return null`? In that case, is there much advantage of doing that over just leaving it without the method and returning `undefined`?

Reply:
I see, that's a fair point, didn't see that `this.temperatureBreachConfiguration` wasn't even a field on `TemperatureBreach`. I was concerned there was a use case where a colour could be set up in Desktop and there was an expectation of colour changing in `BreachChart`, but now I see that it wouldn't work regardless of this PR haha. All good then 😆 .

Reply:
I'm not sure, it looks like the changes were introduced here: https://github.com/openmsupply/mobile/pull/3302/files.

@joshxg any thoughts?

Reply:
Yeah there was intention to use colour on the config to colour the breach chart. Been scrapped.

The config relation on a breach was also changed to explicitly saved the values instead for ensuring history was kept.

Can be removed. It is also referenced elsewhere (breach chart) which could be removed